src/auth/models.cpp

```diff
@@ -44,7 +44,9 @@ const std::vector<Permission> kPermissionsAll = {Permission::MATCH,       Permis
                                                  Permission::FREE_MEMORY, Permission::TRIGGER,
                                                  Permission::CONFIG,      Permission::STREAM,
                                                  Permission::MODULE_READ, Permission::MODULE_WRITE,
-                                                 Permission::WEBSOCKET,   Permission::TRANSACTION_MANAGEMENT};
+                                                 Permission::WEBSOCKET,   Permission::TRANSACTION_MANAGEMENT,
+                                                 Permission::STORAGE_MODE};
```

Comment:
Do we want to handle changing permission modes as privileges? I think there is a bigger story behind it, with disk storage engine coming on and three Memgraph modes. For example an issue I see: we'll try to implement disk storage engine in a way that whenever Memgraph stays out of RAM it fallbacks on disk -> not sure if this and analytics mode should be handled as privilege. Maybe we can ask buda about authorization for storage modes, what do you think?

Reply:
This is good thinking. I am not sure if this fits this PR, like thinking forward so much as we don't know how yet storage will be implemented. I implemented it this way consistency-wise, I think we can refactor when we implement different storage mode, as we will now what are we refactoring for. I am not sure if that makes sense.

Reply:
I think it makes sense to revise that later :+1: